packages/astro-ghostcms/README.md

```diff
@@ -59,7 +59,31 @@ import GhostCMS from '@matthiesenxyz/astro-ghostcms';
 // https://astro.build/config
 export default defineConfig({
   site: "https://YOUR-DOMAIN-HERE.com"
-  integrations: [GhostCMS()],
+  integrations: [
+    GhostCMS({
+      // Config Options
+      ghostURL: "http://example.com", // Recommended to set here, Can also set in .env as CONTENT_API_URL
+      ThemeProvider: { // Allows you to pass config options to our ThemeProvider if enabled.
```

Comment:
a matter of preferences i guess but i'd expect any property in the options to follow `camelCase`

Reply:
as i said, only a matter of preferences!